e2e/libs/keywords/workload_keywords.py

```
@@ -2,6 +2,7 @@
 import asyncio
 
 from node import Node
+from node.node import Node
```

Ruff

Redefinition of unused `Node` from line 4

(F811)